### PR TITLE
fix issue #3073

### PR DESCRIPTION
**Pull request by gyr**
For openSUSE-release-tools-slsa-build-service package create osrt-slsa user and group using sysusers.d.
Move SLSA config files to slsa directory.

### CODE REVIEW

**Comment by Vogtinator** [comment]
Please use a commit and PR title that summarizes what it does

File: Makefile [hunk 1]
```diff
@@ -26,6 +26,7 @@ install:
 	cp -R config/* $(DESTDIR)$(sysconfdir)/$(package_name)
 	rm $(DESTDIR)$(sysconfdir)/$(package_name)/logrotate
 	install -Dpm0644 config/logrotate $(DESTDIR)$(sysconfdir)/logrotate.d/$(package_name)
+	rm $(DESTDIR)$(sysconfdir)/$(package_name)/osrt-slsa-user.conf
```
Comment (Vogtinator) [inline Makefile:29]: Can be merged with the rm above

Comment (gyr) [reply Makefile]: Fixed: created a slsa directory and move related config files there

**Comment by gyr** [comment]
> Please use a commit and PR title that summarizes what it does

Done

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**